helm/templates/deployment.yaml

```diff
@@ -5,29 +5,49 @@ metadata:
     app.component: {{ .Values.containers.service.name }}
   name: {{ .Values.containers.service.name }}
 spec:
-  replicas: {{ .Values.replicas }}
+  replicas: {{ .Values.replicaCount }}
   minReadySeconds: {{ .Values.minReadySeconds }}
-  strategy: {}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: {{ .Values.maxSurge }}
+      maxUnavailable: {{ .Values.maxUnavailable }}
   template:
     metadata:
       labels:
         app.component: {{ .Values.containers.service.name }}
+      annotations:
+        redeployOnChange: "{{ .Values.containers.service.redeployOnChange }}"
     spec:
-      containers:   
+      containers:
       - name: {{ .Values.containers.service.name }}
         image: {{ if .Values.containers.service.registry }}{{ .Values.containers.service.registry }}/{{ end }}{{ .Values.containers.service.image }}:{{ .Values.containers.service.tag }}
+        {{- if .Values.containers.command }}
+        {{- with .Values.containers.command }}
+          command:
+            {{- toYaml . | nindent 12 }}
+        {{- end }}
+        {{- with .Values.containers.args }}
+          args:
+            {{- toYaml . | nindent 12 }}
+        {{- end }}
+        {{- end }}
         imagePullPolicy: {{ .Values.containers.service.imagePullPolicy }}
         env:
         - name: NODE_ENV
           value: {{ .Values.environment }}
+        - name: POSTGRES_DB
+          value: {{ .Values.postgresDatabase }}
         - name: POSTGRES_USERNAME
           value: {{ .Values.postgresUsername }}
         - name: POSTGRES_PASSWORD
           value: {{ .Values.postgresPassword }}
         - name: MINE_SUPPORT_MESSAGE_QUEUE
```

Comment:
There are now other message queue environment variables that need to be added to the helm chart.  

MINE_SUPPORT_MESSAGE_QUEUE,
MINE_SUPPORT_MESSAGE_QUEUE_USER,
MINE_SUPPORT_MESSAGE_QUEUE_PASS,
MINE_SUPPORR_MESSAGE_QUEUE_PORT

(There is also a spelling mistake in the last one)

Reply:
Added, and fixed spelling.